src/colorizer/ColorRamp.ts

```diff
@@ -12,7 +12,16 @@ export default class ColorRamp {
 
   constructor(colorStops: ColorRepresentation[], type: ColorRampType = ColorRampType.LINEAR) {
     this.colorStops = colorStops.map((color) => new Color(color));
-    const dataArr = this.colorStops.flatMap((col) => [col.r, col.g, col.b, 1]);
+    const dataArr = this.colorStops.flatMap((col) => {
+      // Must use getHex() instead of accessing color properties directly (e.g.
+      // `col.r`) so the returned color is in sRGB and not LinearSRGB color
+      // space. See https://threejs.org/manual/#en/color-management.
+      const hex = col.getHex();
+      const r = ((hex >> 16) & 0xff) / 255;
+      const g = ((hex >> 8) & 0xff) / 255;
+      const b = (hex & 0xff) / 255;
+      return [r, g, b, 1];
```

Comment:
Is there a reason you're doing this manually? I don't know much about this, but I looked at the link and saw they have converters like `.convertLinearToSRGB()`, would that work in this case?

Reply:
Oop, great catch! Updated this code.